NuGet.Docs/Create/Creating-and-Publishing-a-Symbol-Package.md

```diff
@@ -3,12 +3,11 @@
 ## Introduction
 
 Apart from building library/content packages and publishing them to [NuGet.org](http://nuget.org),
-NuGet also supports creating symbol/source packages and publishing them to [SymbolSource.org](http://symbolsource.org).
+NuGet also supports creating symbol/source packages and publishing them to the new [SymbolSource.org](https://tripleemcoder.com/2015/10/04/moving-to-the-new-symbolsource-engine/)service.
```

Comment:
Does myget allow us to put the packages on NuGet.org but push the symbols to the myget server?

From: Xavier Decoster <notifications@github.com>
Reply-To: NuGet/NuGetDocs <reply@reply.github.com>
Date: Tuesday, April 12, 2016 at 2:12 AM
To: NuGet/NuGetDocs <NuGetDocs@noreply.github.com>
Cc: Harikrishna Menon <harikm@microsoft.com>
Subject: Re: [NuGet/NuGetDocs] Updated symbols info (#451)


In NuGet.Docs/Create/Creating-and-Publishing-a-Symbol-Package.md<https://github.com/NuGet/NuGetDocs/pull/451#discussion_r59344603>:

> @@ -3,12 +3,11 @@

>  ## Introduction

>

>  Apart from building library/content packages and publishing them to [NuGet.org](http://nuget.org),

> -NuGet also supports creating symbol/source packages and publishing them to [SymbolSource.org](http://symbolsource.org).

> +NuGet also supports creating symbol/source packages and publishing them to the new [SymbolSource.org](https://tripleemcoder.com/2015/10/04/moving-to-the-new-symbolsource-engine/)service.

as well as MyGet.org (http://docs.myget.org/docs/reference/symbols<https://na01.safelinks.protection.outlook.com/?url=http%3a%2f%2fdocs.myget.org%2fdocs%2freference%2fsymbols&data=01%7c01%7charikm%40microsoft.com%7c32b14670d98345d2ab4308d362b2910a%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=5SvPpZDLxEdLJHB%2f7NdozXgxWNlY1Bo%2blPNhB%2fzQdr8%3d>)

—
You are receiving this because you modified the open/close state.
Reply to this email directly or view it on GitHub<https://github.com/NuGet/NuGetDocs/pull/451/files/5e928baf8cf82de70bdcd7a54f015007c83e1079#r59344603>